starter-code/lib/viking.js

```diff
@@ -19,13 +19,87 @@
 //   FootViking: FootViking
 // };
 
-function Soldier () {}
+//------------------------------------------------------
+// SOLDIER
+//------------------------------------------------------
+function Soldier (health, strength, damage) {
+  this.health = health;
+  this.strength = strength;
 
-function Viking () {}
+  this.attack = function() {
+  };
+  this.receiveDamage = function(damage) {
+    this.health = this.health - damage;
+    return this.health;
+  };
+}
 
-function Saxon () {}
+//------------------------------------------------------
+// VIKING
+//------------------------------------------------------
+function Viking (name, health, strength, damage) {
```

Comment:
Don't need to pass in damage as an argument when you create a new Viking.  That's what the strength attribute is for.